components/Courses/CourseSubsription/StartNowButton.tsx

```diff
@@ -1,32 +1,37 @@
 import { View, Pressable, Text } from "react-native";
 import { useNavigation } from "@react-navigation/native";
-import PropTypes from "prop-types";
+import { Course } from "@/types";
+import { t } from "@/i18n";
+import { MaterialCommunityIcons } from "@expo/vector-icons";
 
-const StartNowButton = ({ course }) => {
-  const navigation = useNavigation();
+interface StartNowButtonProps {
+  course: Course;
+}
 
-  const handlePress = () => {
-    navigation.navigate("CourseOverview", {
-      course: course,
-    });
-  };
+export const StartNowButton = ({ course }: StartNowButtonProps) => {
+  const navigation = useNavigation();
 
   return (
-    <View className="flex items-center justify-center">
+    <View className="w-5/6 pt-6">
       <Pressable
-        onPress={handlePress}
-        className="flex w-80 items-center justify-center rounded-lg bg-primary_custom p-4"
+        className="flex-row items-center justify-center rounded-2xl bg-surfaceDefaultCyan p-4"
+        onPress={() => {
+          // @ts-expect-error Will be refactored when we move to Expo Router
+          navigation.navigate("CourseOverview", {
+            course: course,
+          });
+        }}
       >
-        <Text className="p-1 text-lg font-bold text-projectWhite">
-          Começar agora
+        <Text className="p-1 text-surfaceSubtleGrayscale text-h4-sm-bold">
+          {t("course.get-started")}
         </Text>
+        <MaterialCommunityIcons
+          testID="play-circle-outline"
+          name="play-circle-outline"
+          size={20}
+          color="white"
```

Comment:
Remember to use our new color scheme:

```suggestion
          color={colors.surfaceSubtleGrayscale}
```